packages/esm-patient-registration-app/src/patient-registration/form-manager.ts

```diff
@@ -146,6 +135,65 @@ export default class FormManager {
     return savePatientResponse.data.uuid;
   }
 
+  static async saveRelationships(
+    relationships: Array<RelationshipValue>,
+    savePatientResponse: FetchResponse,
+    abortController: AbortController,
+  ) {
+    return Promise.all(
+      relationships
+        .filter((m) => m.relationshipType)
+        .filter((relationship) => !!relationship.action)
+        .map(({ relatedPersonUuid, relationshipType, uuid: relationshipUuid, action }) => {
+          const [type, direction] = relationshipType.split('/');
+          const thisPatientUuid = savePatientResponse.data.uuid;
+          const isAToB = direction === 'aIsToB';
+          const relationshipToSave = {
+            personA: isAToB ? relatedPersonUuid : thisPatientUuid,
+            personB: isAToB ? thisPatientUuid : relatedPersonUuid,
+            relationshipType: type,
+          };
+
+          switch (action) {
+            case 'ADD':
+              return saveRelationship(abortController, relationshipToSave);
+            case 'UPDATE':
+              return updateRelationship(abortController, relationshipUuid, relationshipToSave);
+            case 'DELETE':
+              return deleteRelationship(abortController, relationshipUuid);
+          }
+        }),
+    );
+  }
+
+  static async saveObservations(
+    obss: Record<string, string>,
+    savePatientResponse: FetchResponse,
+    currentLocation: string,
+    currentUser: Session,
+    config: RegistrationConfig,
+    abortController: AbortController,
+  ) {
+    if (Object.keys(obss).length > 0 && config.registrationObs.encounterTypeUuid) {
+      const encounterToSave = {
```

Comment:
I was building this toward the REST API POST Encounter endpoint by mistake. Use the FHIR API.